docroot/themes/contrib/civictheme/civictheme_library/components/00-base/typography/_index.scss

```diff
@@ -9,3 +9,15 @@
     @include civictheme-typography($name);
   }
 }
+
+.civictheme-content {
+  @include civictheme-content();
```

Comment:
please "unwrap" these styles and place them here as-is without mixins and `civictheme-content` classes.  

Please do not modify `civictheme-content();` - we will remove it as a part of update for another component.

Reply:
So all the styles under this function will be in this scss